docker/persister.docker

```diff
@@ -25,4 +25,4 @@ ENV VIRTUAL_ENV="/usr/src/venv"
 ENV PATH="$VIRTUAL_ENV/bin:$PATH"
 WORKDIR /usr/src/app
 
-CMD ["python", "history/subscriber/persister.py"]
+CMD ["sh", "./docker/falcon-docker-persister-entrypoint.sh", "start"]
```

Comment:
Could you explain a little bit what this new file does and why it was created?
Maybe put some comment in the file about what it does too.

Reply:
This file was created with the sole purpose of running the http server in the background, so the endpoints to get and to change the log level, along with the persister service.

Reply:
@mprevide, I really don't understand wihch is the reason for this file, note that history has the same file, so @juliamourac just keep the standard. At my point of view, it's the container manager (compose, kb8s, etc) responsibility to take care of the container restart and retry mechanism, but in the sake of uniformity MAYBE makes sense this file existence. What do you think?

Reply:
> @mprevide, I really don't understand wihch is the reason for this file, note that history has the same file, so @juliamourac just keep the standard. At my point of view, it's the container manager (compose, kb8s, etc) responsibility to take care of the container restart and retry mechanism, but in the sake of uniformity MAYBE makes sense this file existence. What do you think?

Maybe to maintain uniformity really makes sense.